python/ccxt/base/exchange.py

```diff
@@ -436,6 +436,7 @@ def __init__(self, config={}):
 
         self.session = self.session if self.session or not self.synchronous else Session()
         self.logger = self.logger if self.logger else logging.getLogger(__name__)
+        self.has['loadTimeDifference'] = self.has['fetchTime']
```

Comment:
This isn't working. When you print `exchange.has`, loadTimeDifference isn't a key in it

Reply:
This will need to work to get this PR to compile properly